examples/hello_world/ios/Podfile

```diff
@@ -0,0 +1,75 @@
+# Using a CDN with CocoaPods 1.7.2 or later can save a lot of time on pod installation, but it's experimental rather than the default.
```

Comment:
Hm.  We discussed this offline but I think I'm wrong now.  This is needed because hello_world now uses a plugin.

This makes me think that we should either create a new project, or use some other example/integration_test project.  hello_world is meant to be as simple as possible, and I think we should avoid adding plugins to it.